test/modules/collapseWhitespace.js

```diff
@@ -5,9 +5,12 @@ import maxPreset from '../../lib/presets/max';
 
 describe('collapseWhitespace', () => {
     const html = ` <div>
-            <b> hello  world! </b><a>other link
-	</a>
-	  </div>  `;
+    <p>      Hello world	</p>
+    <pre>   <code>	posthtml    htmlnano     </code>	</pre>
+    <code>	posthtml    htmlnano     </code>
```

Comment:
`code` itself (without `pre`) can be trimmed safely while the whitespace around it should be retained.